src/components/pathway-progress/PathwayProgressPage.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useMemo } from 'react';
+import { useParams } from 'react-router-dom';
+import { Helmet } from 'react-helmet';
+
+import { ErrorPage } from '@edx/frontend-platform/react';
+import {
+  Container,
+} from '@edx/paragon';
+
+import { LoadingSpinner } from '../loading-spinner';
+import { useLearnerPathwayProgressData } from './data/hooks';
+import { SubsidyRequestsContextProvider } from '../enterprise-subsidy-requests';
+import { PathwayProgressContextProvider } from './PathwayProgressContextProvider';
+import { CourseEnrollmentsContextProvider } from '../dashboard/main-content/course-enrollments';
+import PathwayProgressHeader from './PathwayProgressHeader';
+import PathwayRequirements from './PathwayRequirements';
+
+export default function PathwayProgressPage() {
+  const { pathwayUUID } = useParams();
+  const [pathwayProgressDetails, fetchError] = useLearnerPathwayProgressData(pathwayUUID);
+
+  const initialState = useMemo(
+    () => {
+      if (!pathwayProgressDetails) {
+        return undefined;
+      }
+      return pathwayProgressDetails.data;
+    },
+    [pathwayProgressDetails],
+  );
+
+  if (fetchError) {
+    return <ErrorPage message={fetchError.message} />;
+  }
+
+  if (!initialState) {
+    return (
+      <Container size="lg" className="py-5">
+        <LoadingSpinner screenReaderText="loading pathway progress" />
+      </Container>
+    );
+  }
+  const PATHWAY_TITLE = `${initialState.pathwayData.title}`;
+
+  return (
+    <>
+      <Helmet title={PATHWAY_TITLE} />
+      <CourseEnrollmentsContextProvider>
+        <SubsidyRequestsContextProvider>
```

Comment:
This provider is already being called in `AuthenticatedUserSubsidyPage` which is wrapped on all [enterprise page routes](https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/master/src/components/app/EnterpriseAppPageRoutes.jsx#L16). I don't think we need to call it here.